samples/Azure.Analytics.Purview.Account/Generated/PurviewAccountCollections.cs

```diff
@@ -709,9 +709,9 @@ internal HttpMessage CreateGetChildCollectionNamesRequest(string skipToken)
             return message;
         }
 
-        internal HttpMessage CreateGetCollectionPathRequest()
+        internal HttpMessage CreateGetCollectionPathRequest(RequestContext context)
```

Comment:
Let's make this an optional parameter by setting it to null in the method signature.

Reply:
We use common method for creating a request for HLC and LLC. For HLC, we don't add the null signature because it's an internal method. That's why I didn't change it. But if you want to me update it, I can do it but it will also update internal RestClient create request methods for HLC.